Formula/openj9.rb

```diff
@@ -0,0 +1,123 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/eclipse/openj9/archive/refs/tags/openj9-0.26.0.tar.gz", using: :nounzip
```

Comment:
Is there any way to specify the `stage` directory for this? if using tarball from github, the version number is wrong.

Reply:
There is no way to specify a stage directory for the main URL. What exactly is wrong about it?

Reply:
`openj9` uses git revision as it version number in the `java -version` output. Currently the version number is same as the revision number in `openj9-openjdk-jdk`.

Reply:
In that case I think it'll be better to manually specify the version number.

Reply:
Are you planning to fix the version issue?

Reply:
Temporarily fixed by adding openj9 git repo as resource.

Reply:
Why not just use that as the formula `url` then move everything in `buildpath` to the `openj9` subdirectory?

Reply:
Moved now.